nix/haskell-project.nix

```
@@ -62,6 +71,19 @@ in
               ++ builtins.attrValues (config.devShell.extraLibraries p);
           };
       });
+      exes =
+        (name: value:
+          builtins.foldl' (x: y: x // y) { }
+            (builtins.map
+              (x: {
+                "${x}" = {
+                  type = "app";
+                  program = "${finalPackages.${name}}/bin/${x}";
```

For the same reason as https://haskell.flake.page/docker#size

```suggestion
                  program = "${pkgs.haskell.lib.justStaticExecutables(finalPackages.${name})}/bin/${x}";
```